charts/kube-master/charts/node-problem-detector/check_tunnel.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+OK=0
+NONOK=1
+UNKNOWN=2
+
+# TODO: how to check if tunnel is up
```

Comment:
just the rough structure for the script. need to discuss how to actually monitor tunnel status